Assets/Scenes/DaggerfallUnityGame.unity

```diff
@@ -753,9 +753,27 @@ Prefab:
       propertyPath: m_Enabled
       value: 1
       objectReference: {fileID: 0}
-    m_RemovedComponents: []
+    m_RemovedComponents:
+    - {fileID: 114583161180529142, guid: 53a45b46ca97a974f8e677f68c2a85e6, type: 2}
```

Comment:
I don't know what the purpose of this is, i can't find any information.

Reply:
I'm not sure about this one, but it's probably OK.